src/components/ConfigSelect/index.tsx

```diff
@@ -0,0 +1,54 @@
+import React from 'react'
+
+import { 
+  ConfigLabelText,
+  ConfigLableContainer,
+  ConfigMetricWrapper,
+  SwitchContainer,
+  Picker
+} from '../Globals/ConfigMetric';
+
+interface ConfigSelectProps {
+  label: string;
+  unit: string | undefined;
+  pickerItems: Array<{
+    value: string | number;
+  }>
+  isDisabled: boolean;
+  settingsHandler: (key: string, value: string | boolean) => void;
+  selectedValue?: string;
+}
+
+const ConfigSelect = ({
+  label,
+  unit,
+  pickerItems,
+  isDisabled,
+  settingsHandler,
+  selectedValue
+}: ConfigSelectProps ) => {
+  return (
+    <ConfigMetricWrapper>
+    <ConfigLableContainer>
+      <ConfigLabelText>
+        {label}	
+      </ConfigLabelText>
+    </ConfigLableContainer>
+    <SwitchContainer>
+    <Picker
+        enabled={isDisabled}
```

Comment:
this is a bit confusing - `enabled` being equal to `isDisabled`. Either it should be `isEnabled`, or it should be set to the opposite of `isDisabled`, no?